src/conversion.rs

```diff
@@ -174,10 +174,7 @@ pub trait FromPyObject<'source>: Sized {
 
 /// Identity conversion: allows using existing `PyObject` instances where
 /// `T: ToPyObject` is expected.
-impl<'a, T: ?Sized> ToPyObject for &'a T
-where
-    T: ToPyObject,
-{
+impl<'a, T: ?Sized + ToPyObject> ToPyObject for &'a T {
```

Comment:
👍 nice! I think the lifetime can maybe also be anonymous

```suggestion
impl<T: ?Sized + ToPyObject> ToPyObject for &'_ T {
```

Reply:
:+1: